usr/sbin/p11sak/p11sak.c

```diff
@@ -2566,7 +2608,7 @@ static CK_RV generate_asymmetric_key(CK_SESSION_HANDLE session, CK_SLOT_ID slot,
                     p11_get_ckr(rc));
             goto done;
         }
-    } else {
+    } else if(attr_string) {
```

Comment:
This is probably unrelated? If it was missing in one of the other commits in this PR, then please update it there. If it fixes something  that has already been merged, then maybe move it to a separate commit?

Reply:
Made change in the right commit